internal/graph/dotgraph.go

```diff
@@ -481,7 +475,7 @@ func (b *builder) tagGroupLabel(g []*Tag) (label string, flat, cum int64) {
 	if dc != 0 {
 		c = c / dc
 	}
-	return formatTag(min.Value, min.Unit) + ".." + formatTag(max.Value, max.Unit), f, c
+	return min.Name + ".." + max.Name, f, c
```

Comment:
It is not clear from the test changes whether you added a test case which would reproduce the bug before the fix?